### PR TITLE
chore: update contribution guidelines

### PR DESCRIPTION
**Pull request by sojan-official**
# Pull Request Template

## Description

- add instructions on translation contribution

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -11,20 +11,36 @@ Thank you for taking an interest in contributing to Chatwoot. Before submitting
...
-
-- If fixing bug:
-  - If you are resolving a special issue, add `Bug: Fix xxxx` (#xxxx is the issue) in your PR title.
-  - Provide a detailed description of the bug in the PR.
-  - Add appropriate test coverage if applicable.
+### Getting Started 
+
+- Before starting your work, ensure an [issue](https://github.com/chatwoot/chatwoot/issues) exist for it. If not feel free to create one.
+- Add a comment on the issue and wait for the issue to be assigned before you start working on it. 
+  - This helps to avoid multiple people working on similar issues.
```
Comment (pranavrajs) [inline docs/contributing.md:26]: Please fix the indentation

File: docs/contributing.md [hunk 2]
```diff
@@ -11,20 +11,36 @@ Thank you for taking an interest in contributing to Chatwoot. Before submitting
...
-- If fixing bug:
-  - If you are resolving a special issue, add `Bug: Fix xxxx` (#xxxx is the issue) in your PR title.
-  - Provide a detailed description of the bug in the PR.
-  - Add appropriate test coverage if applicable.
+### Getting Started 
+
+- Before starting your work, ensure an [issue](https://github.com/chatwoot/chatwoot/issues) exist for it. If not feel free to create one.
+- Add a comment on the issue and wait for the issue to be assigned before you start working on it. 
+  - This helps to avoid multiple people working on similar issues. 
+- If its complex changes, propose it on the issue and wait for one of the core contributors to approve before going into the implementation.
```
Comment (pranavrajs) [inline docs/contributing.md:27]: If the solution is complex, propose the solution on the issue.

File: docs/contributing.md [hunk 3]
```diff
@@ -11,20 +11,36 @@ Thank you for taking an interest in contributing to Chatwoot. Before submitting
...
-  - If you are resolving a special issue, add `Bug: Fix xxxx` (#xxxx is the issue) in your PR title.
-  - Provide a detailed description of the bug in the PR.
-  - Add appropriate test coverage if applicable.
+### Getting Started 
+
+- Before starting your work, ensure an [issue](https://github.com/chatwoot/chatwoot/issues) exist for it. If not feel free to create one.
+- Add a comment on the issue and wait for the issue to be assigned before you start working on it. 
+  - This helps to avoid multiple people working on similar issues. 
+- If its complex changes, propose it on the issue and wait for one of the core contributors to approve before going into the implementation. 
+  - This helps is shorter turn around times in merging PRs
```
Comment (pranavrajs) [inline docs/contributing.md:28]: helps in?

File: docs/contributing.md [hunk 4]
```diff
@@ -11,20 +11,36 @@ Thank you for taking an interest in contributing to Chatwoot. Before submitting
...
+- For new feature requests, Provide a convincing reason to add this feature. Real-life Business use-cases will be super helpful.
```
Comment (pranavrajs) [inline docs/contributing.md:29]: business small letter`b`

File: docs/contributing.md [hunk 5]
```diff
@@ -11,20 +11,36 @@ Thank you for taking an interest in contributing to Chatwoot. Before submitting
...
-  - Add appropriate test coverage if applicable.
+### Getting Started 
+
+- Before starting your work, ensure an [issue](https://github.com/chatwoot/chatwoot/issues) exist for it. If not feel free to create one.
+- Add a comment on the issue and wait for the issue to be assigned before you start working on it. 
+  - This helps to avoid multiple people working on similar issues. 
+- If its complex changes, propose it on the issue and wait for one of the core contributors to approve before going into the implementation. 
+  - This helps is shorter turn around times in merging PRs
+- For new feature requests, Provide a convincing reason to add this feature. Real-life Business use-cases will be super helpful. 
+- Feel free to join our [discord community](https://discord.gg/cJXdrwS), if need further discussions with the core team.
```
Comment (pranavrajs) [inline docs/contributing.md:30]: if you need

File: docs/contributing.md (hunk 1 above)
Comment (sojan-official) [reply docs/contributing.md:26]: it's intended to be a secondary level list item

**Review by pranavrajs — APPROVED** [review]

**Pull request merged**